runtime/src/main/java/org/corfudb/runtime/view/stream/BackpointerStreamView.java

```diff
@@ -35,6 +39,7 @@
 public class BackpointerStreamView extends AbstractQueuedStreamView {
 
     final StreamOptions options;
+    final AddressSpaceOptions addressSpaceOptions = new AddressSpaceOptions();
```

Comment:
This really needs to be in AddressSpaceOptions and not all over each place where an addressspaceoption is needed